### ISSUE TITLE
[Feature Request]: Is there any kind of authentication to login on panel? How am I able to enable it?

### ISSUE THREAD

**Issue by mehdiMj-ir** [RedisInsight/RedisInsight#1829]
Is there any type of authentication on `RedisInsight` panel?

I tried to build a docker image and it works fine, then I tried to serve it with on Nginx and that works fine too.

I wonder if there is any type of authentication or authorization on panel itself (like pgAdmin web).
![Screenshot_20230312_151002](https://user-images.githubusercontent.com/20270329/224542343-b1f103d8-6d44-42d0-9654-444a5fac3d73.png)

I saw #1181 but it's for login to redis server using LDAP but I wanna authentication for users whom login to `RedisInsight` panel.

Thanks in advanced :)

**Comment by ViktarStarastsenka** [comment]
hi @mehdiMj-ir , there is no dedicated panel in RedisInsight, but you can use a reverse proxy + [Redis ACL](https://redis.io/docs/management/security/acl/) to manage the list of users.

**Comment by mehdiMj-ir** [comment]
> hi @mehdiMj-ir , there is no dedicated panel in RedisInsight, but you can use a reverse proxy + [Redis ACL](https://redis.io/docs/management/security/acl/) to manage the list of users.

Thanks for your kind reply. I saw that on docs too but it would be great if RedisInsight panel has authentication mechanism by itself.

So, should I close this issue or leave it open as a feature request?

**Comment by ViktarStarastsenka** [comment]
@mehdiMj-ir , let's leave it open as a feature request
thank you for the feedback!

**Comment by kissless-bhvr** [comment]
This is indeed a pretty useful feature to have! @ViktarStarastsenka are you aware of any workarounds that make it possible to manage access\permissions within the RedisInsight v2? E.g. to allow certain users to edit keys while having read-only access to other users? Thank you in advance!

**Comment by kiss-less** [comment]
This is indeed a pretty useful feature to have! @ViktarStarastsenka are you aware of any workarounds that make it possible to manage access\permissions within the RedisInsight v2? E.g. to allow certain users to edit keys while having read-only access to other users? Thank you in advance!

**Comment by ViktarStarastsenka** [comment]
@kiss-less , you can use standard Redis ACL - https://redis.io/docs/management/security/acl/

**Comment by kiss-less** [comment]
> @kiss-less , you can use standard Redis ACL - https://redis.io/docs/management/security/acl/

Thanks for the quick reply. So if I understood correctly, the idea is to create an ACL and use the credentials from it in RedisInsight's db configuration, correct? Would it be possible to still have the "default user" enabled in this case?

**Comment by ViktarStarastsenka** [comment]
@kiss-less , yes, you can allow specific commands for users (but please include the `INFO` command since this is the command that is required by RedisInsight to connect to a database), and then connect to databases using these users.
the default user will stay, but you can set a password for this user, if needed
